docker/dropin_env_base/Dockerfile

```diff
@@ -10,7 +12,9 @@ RUN apt update && apt upgrade -y && apt install -y python3 python3-pip python3-d
 
 COPY requirements.txt requirements.txt
 COPY datarobot-mlops-*.jar /opt/jars/
+COPY mlops-agent-*.jar /opt/jars/
 
 ENV DRUM_JAVA_SHARED_JARS=/opt/jars/*
+ENV MLOPS_MONITORING_AGENT_JAR_PATH=/opt/jars/mlops-agent-${DATAROBOT_MLOPS_VERSION}.jar
```

Comment:
This is the environment variable that we use to support the embedded agent case.